### ISSUE TITLE
error: trained model does not exist: /tmp/mozilla_by17061430/sparse-to-dense-master/results/nyudepthv2.input=rgbd.nsample=100.rep=linear.encoder=conv.decoder=upproj.criterion=l1.lr=0.01.bs=16.pretrained=true/model_best.t7	

### ISSUE THREAD

**Issue by wuheng199112068** [fangchangma/sparse-to-dense#20]
error: trained model does not exist: /tmp/mozilla_by17061430/sparse-to-dense-master/results/nyudepthv2.input=rgbd.nsample=100.rep=linear.encoder=conv.decoder=upproj.criterion=l1.lr=0.01.bs=16.pretrained=true/model_best.t7	
Usage: [options]

Sparse-to-Dense Training script
See https://github.com/fangchangma/sparse-to-dense for details

Train Options:
  -dataset            Options: nyudepthv2 | kitti [nyudepthv2]
  -inputType          Options: rgb | rgbd | d | g | gd [rgb]
  -nSample            average number of depth samples [0]
  -criterion          Options: l1 | l2 | berhu [l1]
  -pretrain           use pretrained model [false]
  -rep                Representation of depth. Options: linear | log | inverse [linear]
  -encoderType        Options: conv | channeldrop | depthsep [conv]
  -decoderType        Options: deconv2 | deconv3 | upconv | upproj [upproj]
  -manualSeed         Manually set RNG seed [0]
  -nGPU               Number of GPUs to use by default [1]
  -backend            Options: cudnn | cunn [cudnn]
  -cudnn              Options: fastest | default | deterministic [fastest]
  -gen                Path to save generated files [gen]
  -precision          Options: single | double | half [single]
  -nThreads           number of data loading threads [2]
  -nEpochs            Number of total epochs to run [20]
  -epochNumber        Manual epoch number (useful on restarts) [1]
  -batchSize          mini-batch size (1 = pure stochastic) [16]
  -testOnly           Run on validation set only [false]
  -tenCrop            Ten-crop testing [false]
  -resume             Path to directory containing checkpoint [none]
  -LR                 initial learning rate [0.01]
  -momentum           momentum [0.9]
  -weightDecay        weight decay [0.0001]
  -recomputeBatchNorm recompute batch norm statistics [false]
  -shortcutType       Options: A | B | C []
  -optimState         Path to an optimState to reload from [none]
  -optnet             Use optnet to reduce memory usage [true]
  -nClasses           Number of classes in the dataset [0]

非常感谢你提供的代码，我对lua语言不熟，在运行命令行的时候出了以上的错误，这是为什么？你能够提供一个直接可以运行的实例吗？谢谢，盼复。
